src/main/java/core/basesyntax/AreaCalculable.java

```diff
@@ -0,0 +1,2 @@
+package core.basesyntax;public class AreaCalculable {
```

Comment:
AreaCalculable is defined as a class, but according to the checklist, it should be an interface if its purpose is to define a behavior for calculating area. Also, the name should not use verbs. Consider renaming it to something like 'AreaCalculator' and defining it as an interface.